packages/neo4j-driver/test/internal/transaction-executor.test.js

```diff
@@ -17,7 +17,7 @@
  * limitations under the License.
  */
 
-import { newError, error as err, internal } from 'neo4j-driver-core'
+import { newError, error as err, internal, int } from 'neo4j-driver-core'
```

Comment:
what's `int`?

Reply:
It is a function which creates the custom Integer object from a `number`, `string`, `bigint` or `{ low: 1, high: 1 }`. But i will remove the import because it is not being used in the file.